README

```diff
@@ -448,6 +448,9 @@ Features
     - write_riemann
       Sends data to Riemann, a stream processing and monitoring system.
 
+    - write_sensu
+      Sends data to Sensu a stream processing and monitoring system, via sensu client local TCP socket.
```

Comment:
Sends data to Sensu, a stream processing and monitoring system, via the Sensu client local TCP socket.